src/tests/multihost/alltests/proxy_sleep

```diff
@@ -0,0 +1,4 @@
+auth required pam_exec.so /usr/bin/sleep 100
+password required pam_exec.so /usr/bin/sleep 100
+account required pam_exec.so /usr/bin/sleep 100
+session required pam_exec.so /usr/bin/sleep 100
```

Comment:
Is this config required by all the tests in test_misc_proxy.py (currently added for all tests on line 77) I think it can be a fixture creating the file directly on the client or transported after creation from the script. This is a small file that can be created on the fly instead of maintaining it separately. It should be only applied to test_bz1927195.

Reply:
Changed